recipes/jupytext/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jupytext" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 05203c7efc0c7de6947a401538045a42deddbe0b01ed304905f1c9e293d0e742
+
+build:
+  number: 0
+  entry_points:
+    - jupytext = jupytext.cli:jupytext
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - mock
+    - nbformat >=4.0.0
+    - pip
+    - python
+    - pyyaml
+    - testfixtures
+  run:
+    - mock
```

Comment:
Is this needed at runtime? Same with test fixures

Reply:
@mwouts, same here

Reply:
`jupytext` uses `mock` for substituting notebook read/write when using Jupyter. So indeed `mock` is required at runtime. And test fixtures as well, as `jupytext` command line can test the round trip conversion (using `testfixtures`).